databases/week2/part1.sql

```diff
@@ -0,0 +1,60 @@
+/*  Part 1: Working with tasks
+ Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+Change the title of a task
+Change a task due date
+Change a task status
+Mark a task as complete
+Delete a task
+ */
+USE DATABASE `hyfmodels`;
+
+select *
+from user
+where name like 'Anamika%';
+
+select *
+from status;
+
+select *
+from task;
+
+select * from user_task where user_id = 12;
+select * from user_task where task_id = 1;
+
+/*  Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id */
+insert into task (title, description, created, updated, due_date, status_id)
+values ('read a book', 'Maybe something about parenting?', '2023-01-11 06:14:01', '2023-01-11 06:14',
+        '2023-02-11 06:14:01', 2);
+
+insert into user (name, email, phone)
+values ('Anamika Joseph', 'anamikajoseph0@harvard.edu', '123-456-7890');
+
+insert into user_task (user_id, task_id)
+values (12, 36);
```

Comment:
No clue why you need this..? Do you want to create `user_task `& `user` tables..? They are already created.
all attributes: `title, description, created, updated, due_date, status_id, user_id` can be inserted into table task

Reply:
Initially i have done this part with the hyfmodel2 database, then refactored it with hyfmodels database. So if you see the latest files it won't there.